docs-rst/installation/server-config/security.rst

```diff
@@ -29,27 +30,46 @@ You can use KSQL with a Kafka cluster in |ccloud|. For more information, see :re
 Configuring KSQL for Secured Confluent Schema Registry
 ------------------------------------------------------
 
-The following configuration connects KSQL with the Confluent Schema Registry over HTTPS.
+KSQL can be configured to connect to the Schema Registry over HTTP by setting the
+``ksql.schema.registry.url`` to the Schema Registry's HTTPS endpoint.
+Depending on your security setup, you may also need to supply a trust-store, (e.g. if the Schema
+Registry's SSL certificates are not trusted they the JVM by default), and a key-store, (e.g. if the
+Schema Registry requires mutual authentication).
 
-#. Specify the HTTPS endpoint in the ``ksql.schema.registry.url`` setting in the
-   KSQL server configuration file:
+The following configuration is all that is needed to configure KSQL to communicate with the Schema Registry
```

Comment:
Simplify as `To configure KSQL communication with the Schema Registry over HTTPS, where mutual authentication is not required and the Schema Registry's SSL certificates are trusted by the JVM:`